go/cmd/dolt/commands/log.go

```diff
@@ -125,14 +126,26 @@ func logWithLoggerFunc(ctx context.Context, commandStr string, args []string, dE
 		return 1
 	}
 
-	cs, err := parseCommitSpec(dEnv, apr)
-	if err != nil {
-		cli.PrintErr(err)
-		return 1
+	numLines := apr.GetIntOrDefault(numLinesParam, -1)
+
+	argIsRef := false
+	if apr.NArg() == 1 {
```

Comment:
This logic isn't right, you need to be able to do 

`dolt log -n 1 HEAD~ myTable`

The table arg is always the last one, and restricts the log to only a single table